Tasks/MobileCenterDistribute/task.json

```diff
@@ -98,6 +103,7 @@
         },
         {
             "name": "packParentFolder",
+            "aliases": [ "symbolsPackParentDirectory" ],
```

Comment:
"Pack" seems strange. Perhaps more like the label - symbolsIncludeParentDirectory?

Reply:
Let's change it!